spec/spec_helper.cr

```diff
@@ -25,13 +25,21 @@ macro assert_unauthorized(response)
   fail "expected error response not found" unless found
 end
 
-# Disable all authentication checks by making `DppmRestApi::Actions.has_access?`
-# always return true, then set the authentication back to the default system
-# after yielding to the block.
+module DppmRestApi
+  # Disable all authentication checks by making `DppmRestApi::Actions.has_access?`
+  # always return true, then set the authentication back to the default system
+  # after yielding to the block.
+  def self.without_authentication!
```

Comment:
After thinking more, I think it's would be fine to monkey patch if we explicitly state this is used for specs:
```cr
def DppmRestApi.spec_without_authentication!(&block)
  Actions.access_filter = ->(_context : HTTP::Server::Context, _permissions : Access) { true }
  yield
  Actions.access_filter = ->access_filter(HTTP::Server::Context, Access)
end
```

Reply:
I don't like the phrasing of that. What if we create a `SpecHelpers` module (like we have with `RouteHelpers`), so that it's `SpecHelpers.without_authentication!`?

Reply:
That's also good, as long as the name "spec" or "test" is inside to show that it is only used in spec tests.